telegram/ext/filters.py

```diff
@@ -172,10 +173,6 @@ class variable.
             return a dict with lists. The dict will be merged with
             :class:`telegram.ext.CallbackContext`'s internal dict in most cases
             (depends on the handler).
-
-    Attributes:
-        name (:obj:`str`): Name for this filter.
-        data_filter (:obj:`bool`): Whether this filter is a data filter.
```

Comment:
can you revert this change (and the same in `{Message, Update}Filter`) since these attributes are now being repeated in classes like `filters.Command`.

The `:inherited-members:` option [was added ](https://github.com/python-telegram-bot/python-telegram-bot/pull/2759#discussion_r744147138) since I wanted to document the members of `_ChatUserBaseFilter` such as `(add,remove)_usernames` in its subclasses (`filters.{Chat, User, etc}`) without duplicating them.

Alternatively, it maybe possible to modify `autodoc_skip_member` to exclude this in other classes I think

Reply:
I excluded `{Base, Massage, Update}Filter` from the `:inherited-members:`, that does the trick :)

Reply:
though now those 2 properties aren't documented for `Message, UpdateFilter`.. is that okay with you?

Reply:
Yes. We list the base classes, that should be enough. We don't normalle repeat all the parents attributes/methods/… :)